src/lib.rs

```diff
@@ -507,6 +550,13 @@ impl<'ui> Ui<'ui> {
     }
 }
 
+// Widgets: Selectable / Lists
+impl<'ui> Ui<'ui> {
+    pub fn selectable<'p>(&self, label: ImStr<'p>) -> bool {
```

Comment:
I think I didn't want to use the parameters at the time so I didn't expose them. I've added them now. In C++ ImGui these have default values, it will be nice when Rust adds support for default parameters!